docs/datamodel/scalars/enum.rst

```diff
@@ -19,14 +19,21 @@ Enum
 
         scalar type Color extending enum<Red, Green, Blue>;
 
-    :eql:op:`Casting <CAST>` is required to obtain an
+    Enum values can then be accessed directly:
+
+    .. code-block:: edgeql-repl
+
+        db> SELECT Color.Red IS Color;
+        {true}
+
+    :eql:op:`Casting <CAST>` can be used to obtain an
     enum value in an expression:
 
     .. code-block:: edgeql-repl
 
         db> SELECT 'Red' IS Color;
         {false}
-        db> SELECT <Color>'Red' IS Color;
+        db> SELECT <Color>'Red' IS Color.Red;
```

Comment:
This doesn't seem correct: `Color.Red` isn't a type.  Did you mean `SELECT Color.Red IS Color`?

Reply:
Good catch, fixed